docker-compose.bundle.yml

```diff
@@ -0,0 +1,11 @@
+# Copy file to docker-compose.override.yml to override docker-compose.yml
```

Comment:
lets either have this way of doing it OR the comments in docker-compose.yml. its confusing to do both since they accomplish the same task.

Reply:
We weren't familiar with the sleep infinity command. Now that we know that exists we agree and will remove this file.